markdown/developers/modules.md

```diff
@@ -518,6 +518,35 @@ process {
 
 5. Each output file SHOULD be emitted in it's own channel, along with the `meta` map if provided ( the exception is the versions.yml ).
 
+### Documentation
+
+1. Each module MUST have a `meta.yaml` in the same directory as the `main.nf` of the module itself.
+
+2. Keywords SHOULD be sufficient to make the module findable through research domain, data types, and tool function keywords
+
+   - Keywords MUST NOT just be solely of the (sub)tool name
```

Comment:
not for this PR, but can/should we collect keywords somewhere, so one can check what is out there? Some form of controlled words would be awesome at some point

Reply:
hm I guess you can check the modules webpage, but not very obvious maybe.

Reply:
That's the plan for one of spring cleaning tasks, -  go through them all and try and consolidate/standardise them (controlled words might be tricky though)

Reply:
ah right, I remember we talked about this at some point, but forgot the outcome 😆